src/main.tf

```diff
@@ -43,13 +43,19 @@ resource "azurerm_kubernetes_cluster" "main" {
   }
 
   default_node_pool {
+    #bridgecrew:skip=BC_AZR_GENERAL_97:Skipping `Ensure that AKS use the Paid Sku for its SLA`
+    #bridgecrew:skip=BC_AZR_IAM_3:Skipping `Ensure Azure Kubernetes Service (AKS) local admin account is disabled`
+    #bridgecrew:skip=BC_AZR_KUBERNETES_8:Skipping `Ensure AKS uses disk encryption set`
+    #bridgecrew:skip=BC_AZR_KUBERNETES_3:Skipping `Ensure AKS API server defines authorized IP ranges`
+    #bridgecrew:skip=BC_AZR_KUBERNETES_6:Skipping `Ensure AKS enables private clusters`
     name                        = var.node_groups.default_node_group.name
     vm_size                     = var.node_groups.default_node_group.node_size
     min_count                   = var.node_groups.default_node_group.min_size
     max_count                   = var.node_groups.default_node_group.max_size
     vnet_subnet_id              = var.vnet.data.infrastructure.default_subnet_id
     temporary_name_for_rotation = "${var.node_groups.default_node_group.name}temp"
     enable_auto_scaling         = true
+    max_pods                    = 50
```

Comment:
Why was this added? Have we ensured that the nodes can handle 50 pods? In EKS, the pod limit is based on the amount of ENI's that be attached, which changes based on the instance type.

Reply:
Added in via a bridgecrew suggestion. AKS can handle a lot more than 50: https://learn.microsoft.com/en-us/azure/aks/quotas-skus-regions#service-quotas-and-limits

Reply:
Removed this as it was a breaking change